tests/metrics/test_polygon_metrics.py

```diff
@@ -0,0 +1,20 @@
+from nucleus.metrics import PolygonIOU, PolygonPrecision, PolygonRecall
+from tests.metrics.helpers import (
+    TEST_BOX_ANNOTATIONS,
+    TEST_CONVEX_POLYGON_ANNOTATIONS,
+    perfect_match_test_wrapper,
+)
+
+
+def test_perfect_match_box_metrics():
```

Comment:
Seems like you should test a non-perfect match just to make sure logic is correct?

Reply:
Also these tests should be split up so that things can fail separately.

I would recommend writing a single test called identical_values_result_in_a_value_of_1

And then parametrize it

https://docs.pytest.org/en/latest/how-to/parametrize.html

Reply:
Yes, will add more tests. Thanks for the parametrize suggestion, did not know of this!